Zend/tests/list_001.phpt

```diff
@@ -5,6 +5,8 @@
 
 list($a, list($b)) = array(new stdclass, array(new stdclass));
 var_dump($a, $b);
+[$a, [$b]] = array(new stdclass, array(new stdclass));
+var_dump($a, $b);
```

Comment:
I've not added many [] tests, it's using the exactly same syntax (array_pair_list) in parser and the differences are abstracted away inside AST.
If there would be a difference, I'd add tests, but here they would be just outright redundant.